integrated_channels/canvas/client.py

```diff
@@ -51,9 +51,124 @@ def __init__(self, enterprise_configuration):
         self.expires_at = None
 
     def create_course_completion(self, user_id, payload):  # pylint: disable=unused-argument
```

Comment:
my only gripe with this method is that it is rather long! I would recommend refactoring it out a bit into maybe private methods that do one piece (such as handling assignments)

Reply:
Yup, I had the same thought! Let me do some cleanup to make it a little more readable.